src/pages/store/seulgi.js

```diff
@@ -0,0 +1,139 @@
+import { useState, useEffect } from 'react';
+import './seulgi.scss';
+
+const Store = () => {
+  const [list, setList] = useState([]);
+  // const [listType, setListtype] = useState('');
+
+  useEffect(() => {
+    fetch('/Data/ITEM_LIST.json')
+      .then(res => res.json())
+      .then(function (result) {
+        return setList(result);
+      });
+  }, []);
+
+  return (
+    <div className="Store">
+      <section>
+        <h2>전체상품</h2>
+        <div className="itemCategory">
+          <div className="itemAmount">
+            <b>{list.length}</b>
+            <span>개의 상품이 있습니다.</span>
+          </div>
+          <ItemSort />
+        </div>
+        <div className="itemList">
+          {list.map(list => {
+            return (
+              <Items
+                key={list.id}
+                state={list.state}
+                img={list.itemThumbnail}
+                name={list.itemName}
+                itemPrice={list.price}
+              />
+            );
+          })}
```

Comment:
여기도 변수명 신경써주세요!, 그리고 매개변수자리에서 바로 구조분해해주는것도 좋겠네요
```suggestion
          {list.map(({id, state, ~~~~})=> {
            return (
              <Items
                key={list.id}
                state={list.state}
                img={list.itemThumbnail}
                name={list.itemName}
                itemPrice={list.price}
              />
            );
          })}
```